README.md

```diff
@@ -134,7 +173,7 @@ $ bash install.sh -r client
 
 Check mount point at `/cfs/mountpoint` on `client` node defined in `iplist`. 
 
-Open [http://10.196.0.1:8500](https:/github.com/chubaofs/chubaofs) through a browser for monitoring system(the IP of monitoring system is defined in `iplist`). 
+Open [http://10.196.0.1](https:/github.com/chubaofs/chubaofs) through a browser for monitoring system(the IP of monitoring system is defined in `iplist`). 
```

Comment:
Why link a specified local IP address to project master repository.

Reply:
I only changed the monitor port from 8500 to 80 because it was merged with listen.  Maybe i can change it to http://[console-ip-address] through a browser for console system(the IP of console system is defined in `iplist`).

Reply:
OK. I see.